example_project/example_project_combined.json

```diff
@@ -657,7 +652,9 @@
         "onsite_parking_fraction": 1,
         "template": "Residential IECC 2015 - Customizable Template Sep 2020",
         "geometry_unit_orientation": 90.0,
-        "geometry_unit_aspect_ratio" : 1.0
+        "geometry_unit_aspect_ratio": 1.0,
+        "characterize_residential_buildings_from_buildstock_csv": true,
+        "resstock_buildstock_csv_path": "resources/buildstock.csv"
```

Comment:
Assign these at the feature level so we can avoid assigning them to, e.g., ID=17 which uses `hpxml_directory`.